### PR TITLE
feature/PLAYER-5058 Add Skin.JSON parameter to hide UI

### PR DESCRIPTION
**Pull request by KseniiaSokolovaEpam**
If a parameter "controlBar.enabled"  is false, then we do not show the default control bar.

If function "onTogglePlayPause" is defined in playerParams, then we use this function instead of our "togglePlayPause".

Unit tests passed.

### CODE REVIEW

File: js/components/controlBar.js [hunk 1]
```diff
@@ -992,7 +992,9 @@ var ControlBar = createReactClass({
       'oo-flex-end': this.props.audioOnly && Utils.isIos()
     });
 
-    return (
+    return (this.props.skinConfig &&
```
Comment (ooyala-amohnatkin) [inline js/components/controlBar.js]: isn't it better to return `null` at the beginning of render?

File: tests/html5skin-test.js [hunk 2]
```diff
@@ -521,6 +521,69 @@ describe('Controller', function() {
       expect(controller.state.controlBarVisible).toBe(true);
     });
 
+    describe('test customPlayPause', () => {
+      let spy;
+      let testEvent;
+      let testPlayerState;
+      const onTogglePlayPause = (event, playerState) => {
```
Comment (ooyala-amohnatkin) [inline tests/html5skin-test.js]: I think it makes tests overcomplicated. You implement a method in the test and check it's implementation. Checking that it has been called using `toHasBeenCalledWith` would be enough

Comment (ooyala-amohnatkin) [reply tests/html5skin-test.js]: `const onTogglePlayPause = jest.fn();`

File: tests/html5skin-test.js [hunk 3]
```diff
@@ -521,6 +521,69 @@ describe('Controller', function() {
...
+        testEvent = event;
+        testPlayerState = playerState;
+      };
+
+      beforeEach(function() {
+        spy = sinon.spy(controller.mb, 'publish');
```
Comment (ooyala-amohnatkin) [inline tests/html5skin-test.js]: as we use jest maybe replace it with `jest.fn()`?

File: tests/html5skin-test.js (hunk 2 above)
Comment (KseniiaSokolovaEpam) [reply tests/html5skin-test.js]: \+

File: tests/html5skin-test.js (hunk 3 above)
Comment (KseniiaSokolovaEpam) [reply tests/html5skin-test.js]: \+

File: js/components/controlBar.js (hunk 1 above)
Comment (KseniiaSokolovaEpam) [reply js/components/controlBar.js]: \+
thank you for the notice!

**Pull request merged**